test/template-tests/Expand-AzureRMTemplate.ps1

```diff
@@ -151,17 +151,20 @@ function Expand-AzureRMTemplate
             } else {                
                 $HasCreateUIDefinition = $false
                 $createUiDefinitionFullPath = $null 
-            }
-        
+            }       
 
             #*$FolderFiles (a list of objects of each file in the directory)
             $FolderFiles = 
                 @(Get-ChildItem -Path $templateFolder.FullName -Recurse |
                     Where-Object { -not $_.PSIsContainer } |
                     ForEach-Object {
 
-                        # All FolderFile objects will have the following properties:
                         $fileInfo = $_
+                        if ($fileInfo.DirectoryName -like '*__macosx' -or $fileInfo.DirectoryName -like '*source.temp') {

```

Comment:
any reason why we shouldn't do an exact match on this?  Also, we can remove source.temp from the exemption...